Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Replace_section_break_with_page_break
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Open an existing Word document.
+                using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Docx))
+                {
+                    ReplaceSectionBreakWithPageBreak(document);
+                    //Create file stream.
+                    using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../Sample.docx"), FileMode.Create, FileAccess.ReadWrite))
+                    {
+                        //Save the Word document to file stream.
+                        document.Save(outputFileStream, FormatType.Docx);
+                    }
+                }
+            }
+        }
+        /// <summary>
+        /// Replace the section break with page break in the Word document.
+        /// </summary>
+        /// <param name="document"></param>
+        private static void ReplaceSectionBreakWithPageBreak(WordDocument document)
+        {
+            //Add page break and removes section break by moving the section items to the last section.
```

Comment:
Whether we are moving to last section or first section. Check and update XML comment

Reply:
First section